### ISSUE TITLE
wrong usage information

### ISSUE THREAD

**Issue by aschnell** [openSUSE/trollolo#18]
aschnell@thalassa:/ARVIN/yast2/burndown/team-a> trollolo.ruby2.1 plot 
ERROR: "trollolo.ruby2.1 plot" was called with no arguments
Usage: "trollolo.ruby2.1 plot"

**Comment by cornelius** [comment]
Which version are you using? In git master I get the output:

```
ERROR: "trollolo plot" was called with no arguments
Usage: "trollolo plot SPRINT-NUMBER [--output] [--no-tasks] [--with-fast-lane]"
```

**Comment by aschnell** [comment]
aschnell@thalassa:~> trollolo.ruby2.1 --version
Trollolo: 0.0.3

Does that help? AFAICS it's installed as a gem:

/root/.gem/specs/api.rubygems.org%443/quick/Marshal.4.8/trollolo-0.0.3.gemspec

So not an rpm or from git.

**Comment by cornelius** [comment]
I published Trollolo 0.0.4 now, which should fix this issue. Just update your gem.
